tree/treeplayer/src/TDFInterface.cxx

```diff
@@ -128,6 +148,8 @@ Long_t JitTransformation(void *thisPtr, std::string_view methodName, std::string
       // temporaries converted from TTreeReaderArrays.
       if (usedBranchesTypes[i].find_first_of("std::array_view<") == 0u)
          ss << "const ";
+      // Here we do not replace anything: the name of the parameters of the lambda does not need to be the real
+      // column name, it must be an alias to compile.
       ss << usedBranchesTypes[i] << "& " << usedBranches[i] << ", ";
```

Comment:
...this logic to special-case column aliases here and below could go away if we applied aliases before calling `CallJitTransformation` right? :smile: